lib/src/signals/logic.dart

```diff
@@ -633,7 +633,15 @@ class Logic {
     ].swizzle();
   }
 
-  /// Returns a new [Logic] with width [newWidth] where new bits added are sign
+  ///This function calculate the value of a signal or value
```

Comment:
Let's make this a little more descriptive.  Some important topics to hit:
- It performs an "absolute value" on the signal
- Treats the signal as a two's complement signed number

Also, style-wise, put a space between the `///` and your documentation